devel/python/CMakeLists.txt

```diff
@@ -7,22 +7,23 @@ if (NOT DEFINED PYTHON_EXECUTABLE)
    return()
 endif()
 
+python_module(cwrap REQUIRED )
 
-python_module(numpy 1.7.1)
+python_module(numpy REQUIRED 1.7.1)
```

Comment:
I agree, the addition of the `REQUIRED` keyword was because the `python_module` changed.  The two lines are identical.

That said, I will make it `OPTIONAL`.  The user will get a warning if `numpy` is not installed, but everything (incl. Python) will be built.